docs/examples/webextension/background.js

```diff
@@ -0,0 +1,14 @@
+fetch('https://testpilot.firefox.com/api/metrics/ping/testpilottest', {
+  method: 'POST',
+  mode: 'cors',
+  headers: {'Content-Type': 'application/json'},
+  body: JSON.stringify({
+    "some": "random",
+    "metrics": "ping",
+    "payload": "here"
+  })
+}).catch(e => {
+  console.log('problem sending metrics ping', e);
+}).then(resp => {
+  console.log('metrics ping success', resp);
+});
```

Comment:
How does `.catch().then()` work?
I can't remember if that means if you did catch an error it'd fall through and, you'd get both messages:
- _"problems sending metrics ping"_ AND
- _"metrics ping success"_

Or should we reverse the order so it's `.then().catch()`?